core/src/main/java/tech/pegasys/web3signer/core/FilecoinRunner.java

```diff
@@ -31,17 +47,48 @@
 public class FilecoinRunner extends Runner {
 
   private static final String FC_JSON_RPC_PATH = JSON_RPC_PATH + "/filecoin";
+  private final FilecoinNetwork network;
 
-  public FilecoinRunner(final Config config) {
+  public FilecoinRunner(final Config config, final FilecoinNetwork network) {
     super(config);
+    this.network = network;
+  }
+
+  @Override
+  public ArtifactSignerProvider loadSigners(
+      final Config config, final Vertx vertx, final MetricsSystem metricsSystem) {
+    final AzureKeyVaultSignerFactory azureFactory = new AzureKeyVaultSignerFactory();
```

Comment:
nit: would be nice nice to avoid so much dupe with the other runners. also probably not a big deal either.